Mathlib/Analysis/InnerProductSpace/PiL2.lean

```diff
@@ -612,6 +612,38 @@ protected theorem repr_reindex (b : OrthonormalBasis ι 𝕜 E) (e : ι ≃ ι')
 
 end OrthonormalBasis
 
+section EuclideanSpace
+
+variable (𝕜 ι) [IsROrC 𝕜] [Fintype ι] [DecidableEq ι]
+
+/-- The basis `Pi.basisFun` bundled as an orthormal basis of `EuclideanSpace 𝕜 ι`. -/
+noncomputable def EuclideanSpace.basisFun :
```

Comment:
This basis already exists elsewhere, with a different defeq; I'll try to find it when I'm next at lean!

Reply:
(it's effectively `LinearIsometryEquiv.refl`!)

Reply:
Thanks. I use that now to construct `EuclideanSpace.basisFun`